devsite/devsite/settings.py

```diff
@@ -10,10 +10,16 @@
 import sys
 
 from figures.settings.lms_production import (
-    update_celerybeat_schedule,
     update_webpack_loader,
+    update_celerybeat_schedule,
 )
 
+OPENEDX_RELEASE = os.environ.get('OPENEDX_RELEASE', 'HAWTHORN').upper()
```

Comment:
Probably we should have had something like this:

```
settings/common.py
settings/hawthorn.py
settings/ginkgo.py
```

That would make it more readable.

Reply:
For a newcomer to the code, it would be more readable, but then we would need six environment files and then we'd have to maintain them. Then we would feel the pressure to abstract the common settings, then it gets complicated. I think having only two settings files, one for devsite and one for tests is a fair tradeoff. What could help is to make it clear how settings work in the module docstring

Reply:
We would also need settings files for each of the `test_<platform>.py` setting files. I'm also not entirely convinced shared base settings files is a good pattern. I've seen it lead to unnecessary complexity and timing issues